src/main/scala/com/cloudera/sparkts/models/RegressionARIMA.scala

```diff
@@ -0,0 +1,198 @@
+package com.cloudera.sparkts.models
+
+import breeze.linalg
+import breeze.linalg.{DenseMatrix, DenseVector}
+import com.cloudera.sparkts.stats.TimeSeriesStatisticalTests._
+import org.apache.commons.math3.stat.regression.{OLSMultipleLinearRegression, SimpleRegression}
+
+import scala.language.postfixOps
+
+/**
+ * Created by baddar on 1/14/16.
+ * This object contains different methods to
+ */
+
+object RegressionARIMA {
```

Comment:
Would you be able to add a header comment with a brief description of RegressionARIMA and maybe a link to the wikipedia page or a more in depth description?